### DISCUSSION TITLE
Noscript everywhere on a domain except root/home page?

### DISCUSSION THREAD

**Discussion by superuser-does** [uBlockOrigin/uBlock-issues#1878] [Q&A] [answered]
Hi, there's a website where I want to block JavaScript on every page, except the homepage (so scripts enabled on 'example.com', but not 'example.com/page1' or any other page)

Is there any syntax to achieve this in URL filtering ('My rules' section)?

Thanks!

**Comment by gorhill** [comment, marked as answer]
You can use filters for this:

    ||example.com^$csp=script-src 'none'
    @@||example.com/|$csp=script-src 'none'

This won't cause the `noscript` tags to render though.

**Reply by superuser-does** [reply]
Happy with that & it works, thanks!
